src/core/components/auth/auths.jsx

```diff
@@ -80,12 +80,13 @@ export default class Auths extends React.Component {
                   />
               }).toArray()
             }
-            <div className="auth-btn-wrapper">
+            <div className="auth-btn-wrapper sui-btn-wrapper">
               {
-                nonOauthDefinitions.size === authorizedAuth.size ? <Button className="btn modal-btn auth" onClick={ this.logoutClick }>Logout</Button>
-              : <Button type="submit" className="btn modal-btn auth authorize">Authorize</Button>
+                nonOauthDefinitions.size === authorizedAuth.size
+                  ? <Button className="sui-btn sui-btn--tertiary-lt modal-btn auth" onClick={ this.logoutClick } ><span>Logout</span></Button>
```

Comment:
**Change request**: rename the `auth` class so that it makes sense w/r/t BEM in the context of this button (again, no idea what it's related to or trying to convey)

(PS: I know you didn't introduce the `auth` class, or `authorize` for that matter, but if we're going to stick with them in our new button implementation we should address them)

Reply:
> (PS: I know you didn't introduce the `auth` class, or `authorize` for that matter, but if we're going to stick with them in our new button implementation we should address them)

Absolutely. Will update :)

Reply:
@shokey Seems neither classes are required here by CSS but both are used by cypress. 
```
.get("button.btn.modal-btn.auth.authorize.button")
```
Maybe element IDs would be more appropriate here?